### PR TITLE
Add dash_bootstrap_templates

### PR DESCRIPTION
**Pull request by Anselmoo**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Anselmoo** [comment]
Hi https://github.com/orgs/conda-forge/teams/help-python and https://github.com/orgs/conda-forge/teams/staged-recipes this recipe is ready ! Thank you !

**Comment by Anselmoo** [comment]
@conda-forge/help-python 

@conda-forge/staged-recipes

**Comment by Anselmoo** [comment]
@BastianZim can you take a look, please?

File: recipes/dash-bootstrap-templates/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - dash
+    - dash-bootstrap-components >=1.0.0
+    - dash-core-components >=1.3.1
+    - dash-html-components >=1.0.1
+    - dash-renderer >=1.2.4
+    - dash-table >=4.0.0
+    - numpy
+    - importlib-resources >=5.1.0  # [py<39]
+    - importlib-metadata >=3.4.0  # [py==37]
```
Comment (ocefpaf) [inline recipes/dash-bootstrap-templates/meta.yaml:30]:
```suggestion
    - importlib-resources >=5.1.0 
    - importlib-metadata >=3.4.0 
```

File: recipes/dash-bootstrap-templates/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dash-bootstrap-templates-{{ version }}.tar.gz
+  sha256: 746a3e30a2fd963c00b07cbcc23204d27434f2e2793d73d11ffbc5cde0193610
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```
Comment (ocefpaf) [inline recipes/dash-bootstrap-templates/meta.yaml:14]:
```suggestion
  noarch: python
  number: 0
```

**Review by ocefpaf — COMMENTED** [review]
This recipe can probably be noarch.

**Pull request merged**